monkey/tests/unit_tests/infection_monkey/ransomware/test_ransomware_payload.py

```diff
@@ -48,6 +48,22 @@ def ransomware_payload(ransomware_payload_config, telemetry_messenger_spy):
     return RansomwarePayload(ransomware_payload_config, telemetry_messenger_spy)
 
 
+def test_target_dir_whitespaces_stripped(ransomware_payload_config, telemetry_messenger_spy):
+    path_with_whitespaces = "   path/to/directory    "
+    expected_target_dir = "path/to/directory"
+
+    ransomware_payload_config["encryption"]["directories"][
+        "linux_target_dir"
+    ] = ransomware_payload_config["encryption"]["directories"][
+        "linux_target_dir"
+    ] = path_with_whitespaces
+
+    assert (
+        RansomwarePayload(ransomware_payload_config, telemetry_messenger_spy)._target_dir
```

Comment:
We should avoid testing private variables and methods. Unit tests should test the interface, not the inner workings.

This may be moot, however, as the path on line 52 is valid:
```
ubuntu@appimage:~/testdir/space_test $ mkdir -p " t1/t2/t3 " 
ubuntu@appimage:~/testdir/space_test $ cd !$
cd " t1/t2/t3 "
ubuntu@appimage:~/testdir/space_test / t1/t2/t3 $ 
```
We shouldn't be preventing the user from using valid paths, no matter how obnoxiously they've used spaces..